### ISSUE TITLE
[BUG] JSON schema import fails

### ISSUE THREAD

**Issue by igarashitm** [atlasmap/atlasmap#3781]
**Describe the bug**

JSON schema import as a Document silently fails, even the dropdown stays. Sometimes it succeeds, could be some timing issue?

**To Reproduce**

1. Import JSON schema file as a source Document

**Screenshots**

![atlasmap](https://user-images.githubusercontent.com/265462/154126689-484c8038-68b8-4bf2-b06c-7b8339d17728.gif)

**Comment by igarashitm** [comment]
It worked once I reload the whole page (F5)

**Comment by pleacu** [comment]
I went to my local `atlasmap/standalone` and `mvn clean spring-boot:run` followed by `http://localhost:8585/` in a separate browser window in various sequences and can not get the schema source to fail to load (still using the fix here). I also tried the fully standalone `java -jar ~/git-clone/atlasmap/standalone/target/atlasmap-standalone.ja` as well as the standard storybook atlasmap instance - also unable to get it to fail for me. Perhaps a browser instance issue? `Version 90.0.4430.212 (Official Build) (64-bit)`
